_topic_maps/_topic_map.yml

```diff
@@ -3124,8 +3124,8 @@ Topics:
     Dir: preparing_for_image_based_upgrade
     Distros: openshift-origin,openshift-enterprise
     Topics:
-    - Name: Configuring a shared container directory for the image-based upgrade
-      File: cnf-image-based-upgrade-shared-container-image
+    - Name: Configuring a shared container partition for the image-based upgrade
```

Comment:
I have reached out to Ben and Gaurav and Ben assured me that because the docs were just introduced in 4.16 and we're updating the titles closely after, we should be okay to go ahead without redirects.

Reply:
Thanks for checking!